pkg/cmd/server/api/validation/oauth.go

```diff
@@ -78,7 +79,7 @@ func ValidateIdentityProvider(identityProvider api.IdentityProvider) ValidationR
 	} else {
 		switch provider := identityProvider.Provider.Object.(type) {
 		case (*api.RequestHeaderIdentityProvider):
-			validationResults.AddErrors(ValidateRequestHeaderIdentityProvider(provider, identityProvider)...)
+			validationResults.Append(ValidateRequestHeaderIdentityProvider(provider, identityProvider))
```

Comment:
Let's get an issue open to track warnings and errors for authentication config interactions that probably won't work correctly.  Two redirectors or a requestheader challenger and another challenger as a for instance.